soc/intel/intel_ish/intel_ish5/Kconfig.soc

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2023-2024 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config SOC_INTEL_ISH_5_4_1
```

Comment:
previously those 3 have choice wrapped. Why removed?
How to avoid more than one of them are selected?

Reply:
no SoC choice in HWMv2 as its selection is from the chosen board dependency, see also other HWMv2 SoC series, e.g. `soc/gd_gd32/gd32f4xx/Kconfig.soc`